include/aws/auth/signable.h

```diff
@@ -126,13 +126,30 @@ AWS_AUTH_API extern const struct aws_string *g_aws_http_method_property_name;
  */
 AWS_AUTH_API extern const struct aws_string *g_aws_http_uri_property_name;
 
+/**
+ * Name of the property that holds the (hex-encoded) signature value.  This is always added to signing results.
+ */
+AWS_AUTH_API extern const struct aws_string *g_aws_signature_property_name;
+
+/**
+ * Name of the property that holds the (hex-encoded) signature value of the signing event that preceded this one.
+ * This property must appear on signables that represent chunks or events.
+ */
+AWS_AUTH_API extern const struct aws_string *g_aws_previous_signature_property_name;
+
 /*
  * Common signable implementations
  */
 
 AWS_AUTH_API
 struct aws_signable *aws_signable_new_http_request(struct aws_allocator *allocator, struct aws_http_message *request);
 
+AWS_AUTH_API
+struct aws_signable *aws_signable_new_chunk(
```

Comment:
copy/paste the applicable comments/docs you added to `aws_signing_complete_fn` so that they can be found on this function too